pages/linux/caffeinate.md

```diff
@@ -0,0 +1,16 @@
+# caffeinate
+
+> Prevent desktop from sleeping.
+> More information: <https://manned.org/caffeinate>.
```

Comment:
The man page is from this source code https://launchpad.net/caffeine

Reply:
> I think it would make sense to add a new entry for `linux`.
> 
> Reference:
> 
>     * [caffeine](https://manned.org/caffeine)
> 
>     * [caffeinate](https://manned.org/caffeinate)

I used the documentation that @mfrw mentioned in the Issue https://github.com/tldr-pages/tldr/issues/8138

Reply:
> The man page is from this source code https://launchpad.net/caffeine

I think it is better to use manned as I couldn't find the man page over there.